target_ws_pkgs.repos

```diff
@@ -39,9 +39,9 @@ repositories:
     url: https://github.com/ros2/rclc.git
     # NOTE: see the release notes: https://github.com/ros2/rclc/releases/tag/5.0.1
     # especially the PRs touching rosout logging init
-    # NOTE 2: this is two commit past '5.0.1', fixing the clash with 'NONE'.
+    # NOTE 2: this is three commits past '5.0.1', fixing the clash with 'NONE'.
     # See: https://github.com/ros2/rclc/issues/378
-    version: 24ab126bb489984c2b66a72d2ce56e221dd707d1
+    version: 63d5c43993cb2d0d03ecd1b68d771046fd4ad19f
```

Comment:
I would suggest we keep this as it was. The extra commit only changes CI config in `rclc`: https://github.com/ros2/rclc/commit/63d5c43993cb2d0d03ecd1b68d771046fd4ad19f.